Repofile

```diff
@@ -0,0 +1,19 @@
+{
+	"description": "Tradeshift UI is a framework-agnostic JavaScript library to help Tradeshift App developers to create cohesive user experiences and to provide reusable UI components. https://ui.tradeshift.com",
+	"topics": [
+		"ui",
+		"components",
+		"tradeshift",
+		"reusable",
+		"webcomponents",
```

Comment:
I'm not sure that this is a webcomponents library, based on this description: 
https://developer.mozilla.org/en-US/docs/Web/Web_Components

Reply:
Agree! Just copied the current ones :)

Reply:
And this: https://github.com/topics/webcomponents
> Developers can create custom, reusable, encapsulated HTML tags for the web using Web Components. Components use the Custom Element standard, frequently in combination with the Shadow DOM, HTML Templates, and ES Modules.